features/other_a/invite_use.feature

```diff
@@ -29,3 +29,22 @@ I want to use an invitation to create an account
     And I go to SOME_USER2's invitations page
   Then I should be on Scott's user page
     And I should see "Sorry, you don't have permission to access the page you were trying to reach."
+
+  Scenario: Viewing invitation details when the invitee has deleted their account
+    Given the user "creator" exists and is activated
+      And the user "invitee" exists and is activated
+      And an invitation created by "creator" and used by "invitee"
+      And I am logged in as "creator"
+    When I manage invitations belonging to "creator"
```

Comment:
`When I go to creator's manage invitations page` should work without adding any new step definitions:

https://github.com/otwcode/otwarchive/blob/092aa07d6b57ee5cb603a3628b1f0a89b0153783/features/support/paths.rb#L171-L172